src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -189,6 +189,8 @@ message WorkerStatusResponse {
 message FunctionEnvironmentReloadRequest {
   // Environment variables from the current process
   map<string, string> environment_variables = 1;
+  // Current directory of function app
+  string current_directory = 2;
```

Comment:
using old proto file? shouln't this be function_app_directory?

Reply:
oh shoot! thank you!